builder.go

```diff
@@ -475,7 +469,11 @@ func (b *builder) installBootloader(ctx context.Context) error {
 
 func (b *builder) convert2Img(ctx context.Context) error {
 	logrus.Infof("converting to %s", b.format)
-	return exec.Run(ctx, "qemu-img", "convert", b.diskRaw, "-O", b.format, b.diskOut)
+	if b.format == "raw" {
+		return exec.Run(ctx, "cp", b.diskRaw, b.diskOut)
+	} else {
+		return exec.Run(ctx, "qemu-img", "convert", b.diskRaw, "-O", b.format, b.diskOut)
+	}
```

Comment:
```go
if b.format == "raw" {
	if err := os.Rename(b.diskRaw, b.diskOut); err == nil {
		return nil
	}
}
return exec.Run(ctx, "qemu-img", "convert", b.diskRaw, "-O", b.format, b.diskOut)
```

Reply:
maybe it makes most sense to use the existing [MoveFile](https://github.com/linka-cloud/d2vm/blob/main/convert.go#L108) method?

Reply:
You're absolutely right, I'd completely forgotten about this function. 😀